Packs/ContentManagement/ReleaseNotes/1_1_5.md

```diff
@@ -0,0 +1,18 @@
+
+#### Incident Fields
+- **CI/CD Pull Request Reviewer**
+
+#### Playbooks
+##### New: Pull Request Creation - Bitbucket
+- This playbook creates a pull request using Bitbucket integration.
+##### Pull Request Creation - Github
+- Updated the script SuggestBranchName to be generic.
+##### Pull Request Creation - Generic
+- Added the option to create a pull request in Bitbucket.
+
+#### Scripts
+##### New: CommitFilesBitbucket
```

Comment:
This script should be different than Github? 
Is there different functionality?